conf/etc/nginx/sites-available/default

```diff
@@ -63,6 +69,65 @@ server {
 		auth_basic_user_file /opt/piecewise.git/htpasswd;
 	}
 
+	ssl on;
+	ssl_certificate /etc/ssl/localcerts/localcert.pem;
+	ssl_certificate_key /etc/ssl/localcerts/localcert.key;
+
+	ssl_session_timeout 5m;
+
+	ssl_protocols SSLv3 TLSv1 TLSv1.1 TLSv1.2;
+	ssl_ciphers "HIGH:!aNULL:!MD5 or HIGH:!aNULL:!MD5:!3DES";
+	ssl_prefer_server_ciphers on;
+
+}
+
+# Config for http piecewise server. Note that geolocation in Chrome won't work without HTTPS
```

Comment:
Me personal feeling is that it would be better to remove this entire commented out section, which appears to superseded by the changes/additions above. Version control will maintain the history of this in case we want to go back or reference how it used to be.